sycl/unittests/buffer/BufferLocation.cpp

```diff
@@ -71,7 +71,7 @@ static pi_result redefinedDeviceGetInfo(pi_device device,
 
 class BufferTest : public ::testing::Test {
 public:
-  BufferTest() : Plt{sycl::default_selector()} {}
+  BufferTest() : Plt{sycl::default_selector_v} {}
```

Comment:
I think we can drop initialization list here. https://github.com/intel/llvm/pull/6624 makes it unnecessary.